metrics/src/datapoint.rs

```diff
@@ -91,56 +91,40 @@ macro_rules! datapoint {
 #[macro_export]
 macro_rules! datapoint_error {
     ($name:expr) => {
-        if log::log_enabled!(log::Level::Error) {
-            $crate::submit($crate::datapoint!(@point $name), log::Level::Error);
-        }
+        $crate::submit($crate::datapoint!(@point $name), log::Level::Error);
```

Comment:
@pgarg66 that's what you're expecting right? Seems to work as expected.

Reply:
yes, that looks good